src/app/[locale]/projects/detail/[id]/components/ViewFileListIcon.tsx

```diff
@@ -15,11 +16,12 @@ import { IoMdInformationCircle } from 'react-icons/io'
 
 interface Props {
     license: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Why we are disabling the warnings ? `any` is not an error and i do prefer not hide it

Reply:
@heliocastro  I will remove it if allowed to keep the warning

Reply:
@heliocastro I removed it